apps/web/src/features/ratings/components/emoji-rating-popover.test.tsx

```
@@ -141,7 +144,7 @@ describe('EmojiRatingPopover', () => {
     );
 
     expect(screen.getByText('rate with emoji')).toBeInTheDocument();
-    expect(screen.getByText('Test Vibe')).toBeInTheDocument();
+    expect(screen.getByText(/Rate "Test Vibe" with an emoji/)).toBeInTheDocument();
```

ESLint

[error] Replace `screen.getByText(/Rate·"Test·Vibe"·with·an·emoji/)` with `⏎······screen.getByText(/Rate·"Test·Vibe"·with·an·emoji/)⏎····`

(prettier/prettier)